Cargo.toml

```diff
@@ -23,15 +23,19 @@ description = "A standalone implementation of the RedDSA signature scheme."
 features = ["nightly"]
```

Comment:
We'll need `resolver = "2"` in `[package]` above, otherwise cargo uses `std` for `getrandom` due to a dev-dependency

Reply:
Or alternatively, migrate this crate to 2021 edition (doable as MSRV is 1.56), which defaults to the V2 resolver.

Reply:
@str4d good idea, thanks!